pkg/arvo/gen/hood/clay/autocommit.hoon

```diff
@@ -1,4 +1,4 @@
-::  Kiln: Commit from mount
+::  Enable automatic commits to Clay from unix
```

Comment:
"Unix" is capitalized in some of your new comments and not others. I don't have a preference but it should be consistent.